test/blockchain_helpers.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/onflow/cadence/runtime/ast"
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/cadence/runtime/parser"
+	"github.com/onflow/cadence/runtime/sema"
+	"github.com/onflow/cadence/runtime/stdlib"
+)
+
+//go:embed blockchain_helpers.cdc
+var BlockchainHelpers []byte
+
+func BlockchainHelpersChecker() *sema.Checker {
```

Comment:
Where is this injected into the testing framework?

Reply:
@turbolent This is injected in the language server (https://github.com/onflow/cadence-tools/pull/187/files#diff-9dc030944615b63c919527ae35f799e2c63ff5137f18ef2165d07d808fd2b81fR2891-R2895), so that users of vscode-cadence, don't get the `checking of X failed`.

Reply:
If you meant how it is actually imported in the test script environment, then this is the relevant code snippet: https://github.com/onflow/cadence-tools/pull/181/commits/36a65aa824b504c601059b0310223f58ddde8732#diff-2276184d74d31ecfa10f9215cca2f29ff50eabf91d1323130e4b860c236bd9dbR673-R680